CONTRIBUTING.md

```diff
@@ -18,19 +18,32 @@ If you use Visual Studio Code, you can use _Run Build Task_ (Ctrl+Shift+B) to ru
 
 ### Setup
 
-Once you've created a successful (`OK`) build, copy `ver/us/build/` to `ver/us/expected/build/`:
+Once you've created a successful (`OK`) build, run
 
 ```sh
 ./make_expected.sh
 ```
 
-(If you're working with other versions of the game, replace `us` in the file paths.)
+to copy `ver/us/build/` to `ver/us/expected/build/`. (If you're working with other versions of the game, replace `us` in the file paths.)
```

Comment:
We can remove `(If you're working with other versions of the game, replace `us` in the file paths.)` since the script does all versions.

Reply:
LGTM otherwise!